storage/engine/rocksdb_test.go

```diff
@@ -592,3 +592,30 @@ func BenchmarkMVCCComputeStats1Version32Bytes(b *testing.B) {
 func BenchmarkMVCCComputeStats1Version256Bytes(b *testing.B) {
 	runMVCCComputeStats(256, b)
 }
+
+func BenchmarkMVCCPutDelete(b *testing.B) {
+	const cacheSize = 1 << 30 // 1 GB
+
+	stopper := stop.NewStopper()
+	rocksdb := NewInMem(roachpb.Attributes{}, cacheSize, stopper)
+	defer stopper.Stop()
+
+	r := rand.New(rand.NewSource(int64(time.Now().UnixNano())))
+	value := roachpb.MakeValueFromBytes(randutil.RandBytes(r, 10))
+	zeroTS := roachpb.ZeroTimestamp
+	var blockNum int64
+
+	for i := 0; i < b.N; i++ {
+		blockID := r.Int63()
+		blockNum++
+		key := encoding.EncodeVarintAscending(nil, blockID)
+		key = encoding.EncodeVarintAscending(key, blockNum)
+
+		if err := MVCCPut(rocksdb, nil, key, zeroTS, value, txn1); err != nil {
```

Comment:
what's with the `txn1` here? Shouldn't you get a `WriteIntentError` on the `MVCCDelete` which you carry out w/o Txn?